### PR TITLE
Added an action to verify if the LiteSpeed extension is installed

### PR DESCRIPTION
**Pull request by SandakovMM**
The extension is not yet supported in the centos2alma conversion

### CODE REVIEW

File: pleskdistup/actions/extensions.py [hunk 1]
```diff
@@ -164,3 +164,12 @@ def estimate_prepare_time(self) -> int:
 
     def estimate_revert_time(self) -> int:
         return 20
+
+
+class AssertLiteSpeedNotInstalled(action.CheckAction):
```
Comment (kpushkaryov) [inline pleskdistup/actions/extensions.py:169]: This check is single-use. Propose to replace with general `AssertPleskExtensions` (like existing `AssertPleskComponents`).

File: pleskdistup/actions/extensions.py [hunk 2]
```diff
@@ -164,3 +164,12 @@ def estimate_prepare_time(self) -> int:
...
+    def __init__(self):
+        self.name = "checking if LiteSpeed is installed"
+        self.description = """LiteSpeed extension is installed on the system. To proceed with the upgrade, remove the LiteSpeed extension."""
```
Comment (kpushkaryov) [inline pleskdistup/actions/extensions.py:172]: Unneeded triple quotes.

**Review by kpushkaryov — CHANGES_REQUESTED** [review]

File: pleskdistup/actions/extensions.py (hunk 1 above)
Comment (SandakovMM) [reply pleskdistup/actions/extensions.py]: Looks like an overkill, but whatever

File: pleskdistup/actions/extensions.py [hunk 3]
```diff
@@ -164,3 +164,74 @@ def estimate_prepare_time(self) -> int:
...
+    ):
+        if installed is None and not_installed is None:
+            raise ValueError("'installed' and 'not_installed' can't be both None")
+        self.installed = set(installed) if installed is not None else set()
+        self.not_installed = set(not_installed) if not_installed is not None else set()
+        self._name = name
+        self.installed_violations = set()
+        self.not_installed_violations = set()
+        if installed_description is None:
+            self._installed_description = "Required Plesk extensions are missing. To continue, please install the following extensions:\n\t- {installed_violations}"
```
Comment (kpushkaryov) [inline pleskdistup/actions/extensions.py:194]: If `installed_description`, `not_installed_description` aren't `None`, they are not going anywhere. The same bug is in `AssertPleskComponents`.

**Review by kpushkaryov — CHANGES_REQUESTED** [review]

**Review by kpushkaryov — APPROVED** [review]
Please rebase before merge.
Don't forget to update commit messages, because the change set is not about LiteSpeed anymore.

**Pull request merged**